### ISSUE TITLE
[1.3-beta.16] crosswalk doesn't work after upgrade

### ISSUE THREAD

**Issue by crapthings** [meteor/meteor#6480]
from
    1.3-cordova-beta.3

to
    1.3-cordova-beta.5

then tried
    1.3-beta.16

>meteor create androidtest

>meteor add-platform android

>meteor add crosswalk

>meteor run android-device --verbose

	=> Started app on Android Device.
	I20160314-11:46:19.112(8)? E/chromium(28360): [ERROR:runtime_javascript_dialog_manager.cc(69)] Not implemented reached in virtual void xwalk::RuntimeJavaScriptDialogManager::ResetDialogState(content::WebContents*)
	I20160314-11:46:19.183(8)? I/chromium(29139): [INFO:library_loader_hooks.cc(120)] Chromium logging enabled: level = 0, default verbosity = 0
	I20160314-11:46:19.183(8)? E/chromium(29139): [ERROR:xwalk_platform_notification_service.cc(103)] Not implemented reached in virtual bool xwalk::XWalkPlatformNotificationService::GetDisplayedPersistentNotifications(content::BrowserContext*, std::set<std::basic_string<char, std::char_traits<char>, std::allocator<char> > >*)
	I20160314-11:46:19.183(8)? E/chromium(29139): [ERROR:xwalk_browser_context.cc(79)] Failed to read preference, error num: 0
	I20160314-11:46:19.184(8)? E/chromium(29139): [ERROR:unix_domain_server_socket_posix.cc(106)] Not implemented reached in virtual int net::UnixDomainServerSocket::GetLocalAddress(net::IPEndPoint*) const
	I20160314-11:46:19.184(8)? I/chromium(29139): [INFO:xwalk_extension_renderer_controller.cc(47)] EXTENSION PROCESS DISABLED.
	I20160314-11:46:19.184(8)? E/chromium(29139): [ERROR:runtime_javascript_dialog_manager.cc(69)] Not implemented reached in virtual void xwalk::RuntimeJavaScriptDialogManager::ResetDialogState(content::WebContents*)
	I20160314-11:46:19.184(8)? E/chromium(29139): [ERROR:unix_domain_server_socket_posix.cc(106)] Not implemented reached in virtual int net::UnixDomainServerSocket::GetLocalAddress(net::IPEndPoint*) const
	I20160314-11:46:19.184(8)? I/chromium(29780): [INFO:library_loader_hooks.cc(120)] Chromium logging enabled: level = 0, default verbosity = 0
	I20160314-11:46:19.185(8)? E/chromium(29780): [ERROR:xwalk_platform_notification_service.cc(103)] Not implemented reached in virtual bool xwalk::XWalkPlatformNotificationService::GetDisplayedPersistentNotifications(content::BrowserContext*, std::set<std::basic_string<char, std::char_traits<char>, std::allocator<char> > >*)
	I20160314-11:46:19.185(8)? E/chromium(29780): [ERROR:xwalk_browser_context.cc(79)] Failed to read preference, error num: 0
	I20160314-11:46:19.185(8)? E/chromium(29780): [ERROR:unix_domain_server_socket_posix.cc(106)] Not implemented reached in virtual int net::UnixDomainServerSocket::GetLocalAddress(net::IPEndPoint*) const
	I20160314-11:46:19.185(8)? I/chromium(29780): [INFO:xwalk_extension_renderer_controller.cc(47)] EXTENSION PROCESS DISABLED.
	I20160314-11:46:19.185(8)? D/MeteorWebApp(30626): Detected new bundled version, removing versions directory if it exists
	I20160314-11:46:19.186(8)? I/chromium(30626): [INFO:library_loader_hooks.cc(120)] Chromium logging enabled: level = 0, default verbosity = 0
	I20160314-11:46:19.186(8)? E/chromium(30626): [ERROR:xwalk_platform_notification_service.cc(103)] Not implemented reached in virtual bool xwalk::XWalkPlatformNotificationService::GetDisplayedPersistentNotifications(content::BrowserContext*, std::set<std::basic_string<char, std::char_traits<char>, std::allocator<char> > >*)
	I20160314-11:46:19.186(8)? E/chromium(30626): [ERROR:xwalk_browser_context.cc(79)] Failed to read preference, error num: 0
	I20160314-11:46:19.186(8)? E/chromium(30626): [ERROR:unix_domain_server_socket_posix.cc(106)] Not implemented reached in virtual int net::UnixDomainServerSocket::GetLocalAddress(net::IPEndPoint*) const
	I20160314-11:46:19.186(8)? I/chromium(30626): [INFO:xwalk_extension_renderer_controller.cc(47)] EXTENSION PROCESS DISABLED.
	I20160314-11:46:19.186(8)? I/MeteorWebApp(30626): Serving asset bundle with version: cc4a1616274d286003dd2c71b74a5b7144582e22
	I20160314-11:46:19.328(8) (android:http://localhost:12504/864e46d7b10a81b92290ea1242df8ca0fb7a012b.js:7) Uncaught TypeError: Cannot read property 'NODE_ENV' of undefined
	I20160314-11:46:39.005(8)? W/MeteorWebApp(30626): App startup timed out, reverting to last known good version

and node env #6465 

    I20160314-11:46:19.328(8) (android:http://localhost:12504/864e46d7b10a81b92290ea1242df8ca0fb7a012b.js:7) Uncaught TypeError: Cannot read property 'NODE_ENV' of undefined

**Comment by crapthings** [comment]
revert to 1.3-beta.12

cordova-plugin-crosswalk-webview@https://github.com/crosswalk-project/cordova-plugin-crosswalk-webview/tarball/0e8bd0105214becad2e927a1b7cad33d295dfcb8

**Comment by martijnwalraven** [comment]
I believe this is actually a duplicate of #6459, `Uncaught TypeError: Cannot read property 'NODE_ENV' of undefined`, which has been fixed in `1.3-rc.1`.

The Crosswalk log entries are unrelated and are not real errors. The reason they didn't show up before is that I made a change in logging behavior in `1.3-beta.16` in order to show more logs and also show logs from earlier in the startup process. These misleading error messages are something that would have to be fixed in Crosswalk (see https://crosswalk-project.org/jira/browse/XWALK-6033).
